.claude/skills/run-tests/SKILL.md

```
@@ -0,0 +1,82 @@
+
+## macOS Setup for Acceptance Tests
+
+Acceptance tests need Podman with a properly configured machine:
+
+```bash
+./hack/macos/setup-podman-machine.sh   # 4 CPUs, 8GB RAM
+./hack/macos/run-acceptance-tests.sh
+```
+
+Also add to `/etc/hosts`:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)